plugins/src/terminal/mod.rs

```diff
@@ -118,6 +122,15 @@ impl App {
 fn detect_terminal() -> (PathBuf, &'static str) {
     use std::fs::read_link;
 
+    let cosmic_term = Path::new("/usr/bin/cosmic-term");
```

Comment:
It's a little ugly to hardcode /usr/bin. This means installing cosmic-term in a local prefix, or /usr/local/bin or on nix will not trigger this behavior. That probably isn't tooooo bad, but maybe we should just bite the bullet and parse $PATH?

Reply:
Ya, I was thinking so too, but the exiting method of selecting a terminal for the plugin also used /usr/bin. I can change it though.

Reply:
There is code in cosmic-files to detect terminal apps. That may be useful.

Reply:
I've adjusted it to work more like cosmic-files, but I wonder if all terminal apps will be compatible, or if I should ignore the other terminal apps found searching the desktop entries when cosmic-term isn't found. alacritty seems to only support `-e` or `--command`, for example.